evm_loader/program/src/allocator.rs

```diff
@@ -8,7 +8,14 @@ use linked_list_allocator::Heap;
 use solana_program::entrypoint::HEAP_START_ADDRESS;
 use static_assertions::{const_assert, const_assert_eq};
 
-const HEAP_SIZE: usize = 256 * 1024;
+cfg_if::cfg_if! {
+    if #[cfg(feature = "rollup")] {
```

Comment:
You can also use `#[cfg(feature = "rollup")]` and `#[cfg(not(feature = "rollup"))]` instead of `cfg_if::cfg_if!`

Reply:
thanks, agree. But given `cfg_if` is already used in this file, I'd prefer to stick with this pattern, just for consistency.